### PR TITLE
Sync with upstream and remove JMSSerializerBundle

### LINKED ISSUE #48: Attempted to load class "JMSSerializerBundle" error after new installation.
I get the following error after a new CMF installation.

```
ClassNotFoundException in AppKernel.php line 38:
Attempted to load class "JMSSerializerBundle" from namespace "JMS\SerializerBundle".
Did you forget a "use" statement for another namespace?
```
```bash
composer create-project symfony-cmf/standard-edition private '~1.2'

...
symfony-cmf/media-bundle suggests installing jms/serializer-bundle (When using the serializer handler)
...
```

### PR DESCRIPTION
**Pull request by WouterJ**
fix #48

### CODE REVIEW

File: app/config/config.yml [hunk 1]
```diff
@@ -27,44 +35,42 @@ framework:
...
 # Doctrine Configuration (needed for Jackalope Doctrine DBAL)
 doctrine:
     dbal:
-        driver:   "%database_driver%"
-        host:     "%database_host%"
-        port:     "%database_port%"
-        dbname:   "%database_name%"
-        user:     "%database_user%"
-        password: "%database_password%"
+        driver:   pdo_sqlite
```
Comment (dbu) [inline app/config/config.yml]: are you sure we now want to hardcode that?

File: src/AppBundle/Controller/DefaultController.php [hunk 2]
```diff
@@ -4,14 +4,15 @@
...
 {
     /**
-     * @Route("/app/example", name="homepage")
+     * @Route("/", name="homepage")
      */
-    public function indexAction()
-    {
-        return $this->render('default/index.html.twig');
-    }
+    //public function indexAction(Request $request)
```
Comment (dbu) [inline src/AppBundle/Controller/DefaultController.php]: really?

**Comment by dbu** [comment]
i guess we should drop php 5.3 and 5.4

it looks like tests fail because you removed the controller actions.

we get a bunch of warnings, but they come from the bundles.

**Comment by WouterJ** [comment]
Finally!

**Pull request merged**